lib/watchbuild/runner.rb

```diff
@@ -118,16 +117,13 @@ def notify_terminal(build, minutes, url)
     	require 'terminal-notifier'
 
     	TerminalNotifier.notify('Build finished processing',
-                              title: build.app_name,
+                              title: app.name,
                               subtitle: "#{build.train_version} (#{build.build_version})",
                               execute: "open '#{url}'")
     end
 
     def find_build
-      build = nil
-      app.latest_version.candidate_builds.each do |b|
-        build = b if !build || b.upload_date > build.upload_date
-      end
+      build = app.all_processing_builds.sort_by(&:upload_date).last
```

Comment:
I'm not entirely sure if this part works since I don't have a processing build in App Store Connect, so I tested it with `build = app.builds.sort_by(&:upload_date).last` instead